src/browser/modules/App/App.tsx

```diff
@@ -145,6 +146,12 @@ export function App(props: any) {
     props.bus && props.bus.send(initAction.type, initAction)
   }, [props.bus])
 
+  useEffect(() => {
+    window.Canny && window.Canny('initChangelog', cannyOptions)
+    return () => window.Canny && window.Canny('closeChangelog')
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why is this comment needed?

Reply:
@OskarDamkjaer - TSlint is complaining about the dependency list. Any better idea to get rid of it?
<img width="1889" alt="Screenshot 2021-11-17 at 10 27 00" src="https://user-images.githubusercontent.com/26452483/142183417-425cf047-3a0e-4539-8e31-52f4c9e7d0a6.png">

Reply:
Like the warning says -> the window.Canny is an unneeded dependency since mutating it doesn't re-render the component (meaning it's not a prop or state and only those two can go into dependency arrays)

Reply:
Use effect generally has a lot of pitfalls, I don't know if you've read it before, but I do recommend reading this blogpost about it if you're interested https://overreacted.io/a-complete-guide-to-useeffect/

Reply:
So an empty dependency array is the right move for this component + making sure that window.Canny is defined before calling the function

Reply:
Wait now that I think of it. What we should do is move this logic to the `CannyLoader` component in `canny.ts`. It makes a lot more sense that it'd live there _and_ we'd get around the issue of window.canny sometimes not having been loaded yet!